QuizWhois.Domain/Services/Implementations/QuizService.cs

```diff
@@ -57,6 +59,37 @@ public async Task AddToQuiz(AddToSetModel addToSetModel)
             _logger.LogInformation(messageToLog);
         }
 
+        public QuizModel GetQuiz(long quizId)
+        {
+            if (quizId <= 0)
+            {
+                throw new Exception("Id was invalid number");
+            }
+
+            var entity = _dbContext.Quizzes.Where(x => x.Id == quizId).Select(x => new
+            {
+                Id = x.Id,
```

Comment:
Lets discuss using automaper? Or at least do the mapping in extension method Map() or something alike

Reply:
later